### PR TITLE
Fix keras Conv2D BiasAdd fuse

### PR DESCRIPTION
**Pull request by hwangdeyu**
fixes https://github.com/onnx/tensorflow-onnx/issues/1741.

Signed-off-by: hwangdeyu <dejack953@outlook.com>

### CODE REVIEW

File: tf2onnx/rewriter/__init__.py [hunk 1]
```diff
@@ -49,6 +48,7 @@
     "rewrite_quantize_and_dequantize",
     "rewrite_layer_normalization",
     "rewrite_conv_dilations",
+    "rewrite_conv2d_with_pad",
```
Comment (hwangdeyu) [inline tf2onnx/rewriter/__init__.py:51]: `rewrite_biasadd_with_conv2d` fuses change the tf op `Conv2D` to ONNX op 'Conv'.
So `rewrite_conv2d_with_pad` should be set after `rewrite_biasadd_with_conv2d`.

File: tf2onnx/convert.py [hunk 2]
```diff
@@ -373,6 +373,10 @@ def _from_keras_tf1(model, input_signature=None, opset=None, custom_ops=None, cu
 
     with tf.device("/cpu:0"):
         frozen_graph, initialized_tables = tf_loader.freeze_session(sess, input_names, output_names, get_tables=True)
+        tf_loader.tf_reset_default_graph()
+        tf.import_graph_def(frozen_graph, name='')
+        sess = tf.keras.backend.get_session(model.outputs)
+        frozen_graph = tf_loader.tf_optimize(input_names, output_names, frozen_graph)
```
Comment (hwangdeyu) [inline tf2onnx/convert.py:379]: fixes https://github.com/onnx/tensorflow-onnx/issues/1792.

**Comment by hwangdeyu** [comment]
Hi @guschmue, 
after adding the `tf_optimize()` in `_from_keras_tf1', the optimize() may make it model change. So all [the keras test](https://dev.azure.com/tensorflow-onnx/tensorflow-onnx/_build/results?buildId=16516&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642) models using model predict are failed. 
Do you have any idea to fix this cases?

**Comment by guschmue** [comment]
tf_optimize() sure will change the graph but should not change numerical results. We call it for every path (just not the one in this issue) - must be some problem how we call it in this case.

**Comment by hwangdeyu** [comment]
> tf_optimize() sure will change the graph but should not change numerical results. We call it for every path (just not the one in this issue) - must be some problem how we call it in this case.

Yeah, there may have been a misunderstanding in my comment before.

I mean that after adding the optimize change code, some tests are failed in [ `model.predict test_api.py#L68` ](https://github.com/hwangdeyu/tensorflow-onnx/blob/23b67120e510842da4658ebcae90856b8c49dcd6/tests/test_api.py#L68) and throw the error:
```
tensorflow.python.framework.errors_impl.NotFoundError: PruneForTargets: Some target nodes not found: predict/group_deps 
```
But [test_api.py #L65 L66](https://github.com/hwangdeyu/tensorflow-onnx/blob/23b67120e510842da4658ebcae90856b8c49dcd6/tests/test_api.py#L65) can be passed. So the onnx graph is ok and onnxruntime can be run successfully and get the same numerical results with original keras model.
Now I'm tring to figure out how the `tf_optimize()` change the graph and make model predict failed.

File: tests/test_backend.py [hunk 3]
```diff
@@ -2353,6 +2376,9 @@ def func(x):
             return tf.identity(x_, name=_TFOUTPUT)
         self._run_test_case(func, [_OUTPUT], {_INPUT: x_val})
 
+    @skip_tflite("skip_tflite is not support unit32 when tf version <= 2.3.0")
```
Comment (fatcat-z) [inline tests/test_backend.py:2379]: ... does not support unit32 if tf version <= 2.3.0.

Comment (hwangdeyu) [reply tests/test_backend.py:2379]: Fixed!

**Pull request merged**